src/components/DetailsView/DetailsView.tsx

```diff
@@ -30,32 +47,30 @@ function DetailsView() {
 
   return (
     <div>
-      <img
-        src={data.image}
-        alt={data.fullName}
-        className="block rounded-lg object-none"
-      ></img>
+      <div className="rounded-lg">
+        <Image
+          src={data.image}
+          width={350}
+          height={500}
+          alt={data.fullName}
+          style={imageStyle}
```

Comment:
Why is borderRadius: '10px' defined in a separate imageStyle constant? Could it be added directly in style={...} or moved to CSS/Tailwind for consistency?
```suggestion
         style={{borderRadius: '10px'}} 
```
or
```suggestion
          className='rounded-lg'
```